docs/en/operations/system_tables.md

```diff
@@ -97,19 +107,19 @@ Contains information about external dictionaries.
 
 Columns:
 
-- `name String` — Dictionary name.
-- `type String` — Dictionary type: Flat, Hashed, Cache.
-- `origin String` — Path to the configuration file that describes the dictionary.
-- `attribute.names Array(String)` — Array of attribute names provided by the dictionary.
-- `attribute.types Array(String)` — Corresponding array of attribute types that are provided by the dictionary.
-- `has_hierarchy UInt8` — Whether the dictionary is hierarchical.
-- `bytes_allocated UInt64` — The amount of RAM the dictionary uses.
-- `hit_rate Float64` — For cache dictionaries, the percentage of uses for which the value was in the cache.
-- `element_count UInt64` — The number of items stored in the dictionary.
-- `load_factor Float64` — The percentage filled in the dictionary (for a hashed dictionary, the percentage filled in the hash table).
-- `creation_time DateTime` — The time when the dictionary was created or last successfully reloaded.
-- `last_exception String` — Text of the error that occurs when creating or reloading the dictionary if the dictionary couldn't be created.
-- `source String` — Text describing the data source for the dictionary.
+- `name` (String) — Dictionary name.
```

Comment:
Let's merge the unrelated documentation change as another PR.

Reply:
Ok, reverted that back, leaving only relevant documentation changes.